rust/private/rustc.bzl

```diff
@@ -459,7 +459,8 @@ def construct_arguments(
     # These always need to be added, even if not linking this crate.
     add_crate_link_flags(args, dep_info)
 
-    if crate_info.type == "proc-macro" and crate_info.edition != "2015":
+    is_type_proc_macro = crate_info.type == "proc-macro" or crate_info.wrapped_crate_type == "proc-macro"
```

Comment:
Nit: rename `-is_type_proc_macro` to `needs_extern_proc_macro_flag. Please move edition check there too, so the if below is only

```
if needs_extern_proc_macro_flag:
```

Reply:
done